.evergreen/resync-specs.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+# exit when any command fails
+set -e
+PYMONGO=~/Work/mongo-python-driver
+SPECS=~/Work/specifications
```

Comment:
Any way we can make it so we don't have to pass the `-s` flag for the specs repo on every run? Perhaps:
```
MDB_SPECS=${MDB_SPECS:-~/Work/specifications}
```

Then we can each configure the MDB_SPECS environment variable to point to our own folder.

Reply:
Done.